### ISSUE TITLE
Set LANG and friends

### ISSUE THREAD

**Issue by stdweird** [quattor/ncm-ncd#54]
Some user got

```
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
LANGUAGE = (unset),
LC_ALL = (unset),
LC_CTYPE = "UTF-8",
LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
```

**Comment by jrha** [comment]
Severity?
Likelihood of fix for 16.10?

**Comment by stdweird** [comment]
https://github.com/quattor/ncm-ncd/issues/85 was fixed by setting set the initial ENV to only PATH (and leaving LANG and friends to the system defaults).

this probably reduces the reported issue to possible misconfigured default system settings.
if needed, we can always set LANG like the example environ reported in the issue https://github.com/quattor/ncm-ncd/issues/85#issuecomment-251591518
